code/control/FreshdeskSSO.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+class FreshdeskSSO extends Controller
```

Comment:
We might need to spin this out if we'll want to pull this into some other places - we don't want to pull in userforms without a need for it.

Maybe there is a need for platform-sso module that brings all those kind of things together?